fighthealthinsurance/admin.py

```
@@ -735,3 +737,235 @@ def has_change_permission(self, request, obj=None):
+            if obj.encrypted_blob:
+                try:
+                    obj.encrypted_blob.delete(save=False)
+                except Exception as e:
+                    logger.warning(
+                        f"Failed to delete encrypted_blob for FuzzAttempt {obj.id}: {e}"
+                    )
+        queryset.delete()
+        self.message_user(request, f"Purged {count} fuzz attempt records.")
+
+    @admin.action(description="Export metadata as CSV")
+    def export_metadata_csv(self, request, queryset):
```

Ruff (0.15.1)

[warning] Unused method argument: `request`

(ARG002)